vendor/engines/bulk_email/app/models/bulk_email/facility_extension.rb

```diff
@@ -0,0 +1,17 @@
+module BulkEmail
+
+  module FacilityExtension
```

Comment:
I'm not opposed to it, but do we really need this extension? Could we get away with searching `BulkEmail::Job.where(facility: current_facility)` when we need it? I'm just wondering if it's worth the additional footprint on Facility.